### PR TITLE
Add switch camera button to mobile device onboarding

### PR DESCRIPTION
**Pull request by ryaplots**
#### Summary

This PR implements a camera switch button for mobile devices in the device onboarding.

#### Changes

- Add switch camera button
- Add switch camera callback

#### Testing

1. Log in the console on a mobile device
2. Go to register end device
3. Click on scan end device QR code
4. See the switch button at the bottom of the video
5. Click and see the camera changing

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [x] The steps/process to test this feature are clearly explained including testing for regressions.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: pkg/webui/components/qr/input/video/index.js [hunk 1]
```diff
@@ -43,16 +47,21 @@ const Video = props => {
     if (devices.length && !stream) {
       const ua = navigator.userAgent.toLowerCase()
       const cameras = devices.filter(device => device.kind === 'videoinput')
+      setCameras(cameras)
+      let rearCamera = cameras.find(device => device.label.toLowerCase().includes('back'))
+      if (!rearCamera) {
+        rearCamera = cameras[1]
```
Comment (kschiffer) [inline pkg/webui/components/qr/input/video/index.js]: I suspect this would crash if the camera only has one camera (?)

**Review by kschiffer — COMMENTED** [review]
Can you also include the following change: 

```diff
diff --git a/pkg/webui/components/qr/input/video/video.styl b/pkg/webui/components/qr/input/video/video.styl
index 6d2f701caa..f771df20ec 100644
--- a/pkg/webui/components/qr/input/video/video.styl
+++ b/pkg/webui/components/qr/input/video/video.styl
@@ -16,7 +16,10 @@
   width: 100%
 
 .container
-  max-height: 40vh
+    +media-query($bp.s)
+      max-height: 75vh
+    +media-query-min($bp.s)
+      max-height: 40vh
   overflow: hidden
   display: flex
   align-items: center
```

File: pkg/webui/components/qr/input/video/index.js (hunk 1 above)
Comment (ryaplots) [reply pkg/webui/components/qr/input/video/index.js]: it's how it was before on line 50. This bit: [https://github.com/TheThingsNetwork/lorawan-stack/blob/8eab0b5275e6cb7593dbb01fdea4574ce76ef252/pkg/webui/components/qr/input/video/index.js#L50](https://github.com/TheThingsNetwork/lorawan-stack/blob/8eab0b5275e6cb7593dbb01fdea4574ce76ef252/pkg/webui/components/qr/input/video/index.js#L50)

**Comment by ryaplots** [comment]
> Can you also include the following change:

I will, when merging this to TTSE. the file `pkg/webui/components/qr/input/video/video.styl` does not exist on TTSOS

**Comment by kschiffer** [comment]
That's odd. It seems like `video.styl` is not used anywhere, instead it's `qr.styl`. So can you remove `video.styl` from TTSE then and apply the following:
```diff
diff --git a/pkg/webui/components/qr/qr.styl b/pkg/webui/components/qr/qr.styl
index f0c45f1d40..6eb2b3d991 100644
--- a/pkg/webui/components/qr/qr.styl
+++ b/pkg/webui/components/qr/qr.styl
@@ -14,7 +14,6 @@
 
 .wrapper
   width: 100%
-  aspect-ratio: 4 / 3
   background: black
   display: flex
   flex-direction: column
@@ -32,7 +31,7 @@
 
 .video
   width: 100%
-  aspect-ratio: 4 / 3
+  max-height: 75vh
 
 .msg
   color: #ffffff
```

**Review by kschiffer — CHANGES_REQUESTED** [review]
See above.

**Review by kschiffer — APPROVED** [review]

**Comment by KrishnaIyer** [comment]
This PR is missing a changelog. I've added this here https://github.com/TheThingsNetwork/lorawan-stack/pull/6522

**Pull request merged**